### DISCUSSION TITLE
Github Octernship Eligibility

### DISCUSSION THREAD

**Discussion by hnhparitosh** [community/community#62678] [GitHub Education]
### Hi everyone,

Hi! I am about to graduate soon, but my verified student status on Github Campus will continue for a few more months after my graduation. So am I eligible to apply and continue with the Octernships after I graduate and before my verification on Github Campus expires?

As per this [link by Github](https://docs.github.com/en/education/explore-internship-projects-with-github-octernships/applying-for-github-octernships) :
To apply for internships, you must meet the following requirements:

- You have been accepted to GitHub Global Campus. For more information, including how to apply, see "[Apply to GitHub Global Campus as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)."
- You are at least 18 years old.
- You have an active GitHub account.
- You are a resident of one of the countries where GitHub Octernships is available.

**Comment by Shr11** [comment]
surely! I believe as long as you meet these requirements , none of the organizations ask you about your year of college but Github Global Campus is a must though..

**Comment by codecommander03** [comment]
As long as your verified student status doesn't expire you are good to go.
# Suggestion:
<ul>
 <li> Be familiar with github (can do github learning paths)
 <li> Be an active contributor
</ul>
